nemo/collections/asr/parts/submodules/spectr_augment.py

```diff
@@ -56,7 +60,14 @@ def output_types(self):
         return {"augmented_spec": NeuralType(('B', 'D', 'T'), SpectrogramType())}
 
     def __init__(
-        self, freq_masks=0, time_masks=0, freq_width=10, time_width=10, rng=None, mask_value=0.0,
+        self,
+        freq_masks: int = 0,
+        time_masks: int = 0,
+        freq_width: int = 10,
+        time_width: int | float = 10,
+        rng: random.Random | None = None,
+        mask_value: float = 0.0,
+        fast: bool = True,
```

Comment:
Give flag a proper name

Reply:
... vectorized?

Reply:
I changed it and it gives error on a test. Doing _grep -R rng_ inside the nemo/ folder shows that the _rng_ variable is used everywhere to mean random generator, so I think it is better to leave as it is for consistency with the rest of the project.

Reply:
Never mind, I realized that I was looking at the wrong flag.

Reply:
use_vectorized_version ?

Reply:
or use_vectorized_code

Reply:
I like use_vectorized_code

Reply:
ok. I used that one.